.github/workflows/check_docs_build.yml

```diff
@@ -28,4 +28,6 @@ jobs:
 
       - name: Build documentation
         run: |
+          pip install -e . --no-dependencies
+          pip install torch
```

Comment:
Why are these dependencies needed for building documentation?

Reply:
Because torch is imported somewhere in the codebase and `sphinx` raises a warning about this